app/controllers/videos_controller.rb

```diff
@@ -1,9 +1,10 @@
 class VideosController < ApplicationController
   after_action :verify_authorized, except: %i[index]
   before_action :set_cache_control_headers, only: %i[index]
-  before_action :authorize_video, only: %i[new create]
```

Comment:
the counter-argument is that having these declared in the before actions makes it less likely to add what should have been an authorized action and mistakenly leave it unauthorized. 

The only: list makes that point moot (since any new action either needs to authorize first, or needs to be added to the list, or we're back where we started.